MyBlog.Services/MyBlog.Services.csproj

```diff
@@ -11,4 +11,12 @@
       <ProjectReference Include="..\MyBlog.Infrastructure\MyBlog.Infrastructure.csproj" />
     </ItemGroup>
 
+    <ItemGroup>
+      <Compile Remove="Areas\Users\Mapper\Dto\UserDtoInput.cs" />
+    </ItemGroup>
+
+    <ItemGroup>
+      <PackageReference Include="AutoMapper" Version="12.0.1" />
+    </ItemGroup>
+
 </Project>
```

Comment:
<ItemGroup>
      <Compile Remove="Areas\Users\Mapper\Dto\UserDtoInput.cs" />
    </ItemGroup>

    <ItemGroup>
      <PackageReference Include="AutoMapper" Version="12.0.1" />
    </ItemGroup>
    
    you can remove this lines, it`s not important

Reply:
If i delete this lines, my project asks "You need instaled package ...".